optionals/compat_gm/XEH_preinit.sqf

```diff
@@ -6,4 +6,9 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+// handle GM uniform additional insignia
+[QEGVAR(arsenal,displayClosed), {
+    ACE_arsenal_center call gm_core_characters_fnc_updateUniformDetails;
```

Comment:
maybe check just in-case the unit became null?

```suggestion
    if (alive EGVAR(arsenal,center)) then { EGVAR(arsenal,center) call gm_core_characters_fnc_updateUniformDetails; };
```

Reply:
The GM function does config checks if it can even run on the unit, in case of null it will get empty class so it should early exit, not sure if needed.